scripts/cat/status.py

```diff
@@ -357,6 +357,13 @@ def is_other_clancat(self) -> bool:
     def is_leader(self) -> bool:
         return self.rank == CatRank.LEADER
 
+    @property
+    def joined_group_this_moon(self) -> bool:
```

Comment:
Perhaps rename to `did_join_group_this_moon` instead.
(More consistent with `is_`, `has_`, `do_`-nomenclature for boolean-members)